common/locales/en/generic.json

```diff
@@ -76,6 +77,8 @@
     "gems": "Gems",
     "gemButton": "You have <%= number %> Gems.",
     "moreInfo": "More Info",
+    "moreInfoChallengesLink": "http://habitica.wikia.com/wiki/Challenges",
+    "moreInfoTagsLink": "http://habitica.wikia.com/wiki/Tags",
```

Comment:
Same nit/suggestion for these two strings regarding link vs url.  Feel free to ignore.